### PR TITLE
Added pages and css styles to the sidebar

### PR DESCRIPTION
**Pull request by saidatatanda**
image

### CODE REVIEW

File: ims_app/src/components/screens/inventories/inventories.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import React, { Component } from 'react';
 class Inventories extends Component {
     render() {
         return (
-            <div className='page'>
+            <div className='categories'>
```
Comment (damlois) [inline ims_app/src/components/screens/inventories/inventories.js]: why did you rename this to categories?

File: ims_app/src/components/screens/purchases/purchases.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+
+
+class Purchases extends Component {
+    render () {
+        return (
+            <div className = "purchases">
```
Comment (damlois) [inline ims_app/src/components/screens/purchases/purchases.js]: name the class, page.

File: ims_app/src/components/screens/sales/sales.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+
+
+class Sales extends Component {
+    render () {
+        return (
+            <div className = "sales">
```
Comment (damlois) [inline ims_app/src/components/screens/sales/sales.js]: change name to page

File: ims_app/src/components/screens/vendors/vendors.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+
+
+class Vendors extends Component {
+    render () {
+        return (
+            <div className = "vendors">
```
Comment (damlois) [inline ims_app/src/components/screens/vendors/vendors.js]: also here.

File: package-lock.json [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "dufuna-fem-hackathon-team3",
```
Comment (damlois) [inline package-lock.json]: why do we have this new package.lock.json file?

File: package.json [hunk 6]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (damlois) [inline package.json]: and this too? if they are not doing anything, you should remove them.

File: ims_app/src/components/screens/inventories/inventories.js (hunk 1 above)
Comment (saidatatanda) [reply ims_app/src/components/screens/inventories/inventories.js]: It must have been a mistake. I have effected the correction

File: package.json (hunk 6 above)
Comment (saidatatanda) [reply package.json]: Those files were created after running 'npm install' on my system.

File: ims_app/src/assets/icons/Categories.svg [hunk 7]
```diff
@@ -1,3 +1,3 @@
-<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (damlois) [inline ims_app/src/assets/icons/Categories.svg:1]: are you using this icon?
